drivers/sensor/bmm150/Kconfig

```diff
@@ -0,0 +1,81 @@
+# Kconfig - BMM150 Geomagnetic sensor configuration options
+
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BMM150
+	bool "BMM150 I2C Geomagnetic Chip"
+	depends on SENSOR && I2C
+	default n
+	help
+	  Enable driver for BMM150 I2C-based Geomagnetic sensor.
+
+config BMM150_DEV_NAME
+	string "BMM150 device name"
+	depends on BMM150
```

Comment:
remove all the "depends on BMM150" in this file, the if BMM150 above will replace those.